src/blocks/Tabs/Tabs.tsx

```diff
@@ -80,6 +80,7 @@ export const TabsBlock = ({
 
     const showMedia = Boolean(activeTabData?.media || imageProps);
     const showText = Boolean(activeTabData?.text);
+    const border = activeTabData?.border || 'shadow';
```

Comment:
Are we have enum for this value `shadow`?

Reply:
no, we haven't